package.json

```diff
@@ -12,7 +12,7 @@
 		"seed": "node ./node_modules/knex/bin/cli.js seed:run",
 		"seed:test": "node ./node_modules/knex/bin/cli.js --env test seed:run",
 		"setup_db": "npm run migrate && npm run migrate:test && npm run seed && npm run seed:test",
-        "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha \"src/test/cases/**/*.js\""
```

Comment:
Is this just a linting whitespace change?

Reply:
Yeah that's just a whitespace change.